code/game/machinery/autolathe_datums.dm

```diff
@@ -456,3 +456,55 @@
 	name = "miniature cell"
 	path = /obj/item/weapon/cell/device/emergency_light/empty
 	category = "Engineering"
+	
+/datum/autolathe/recipe/cable_coil
+	name = "cable coil"
+	path = /obj/item/stack/cable_coil
+	category = "Devices and Components"
+	is_stack = 1
```

Comment:
This should give you stacks of cable coil

Reply:
There was a problem where it would give a load of other options, which past x80 wouldn't work. There was the 'Update autolathe.dm' commit that should change it to a maximum x30 based on the fix bay used, I tried it out and took a screenshot.
![autolathefixed](https://user-images.githubusercontent.com/30562547/53349645-b24d0800-3915-11e9-926f-d12a3877627e.png)
Originally it was like this...
![autolatheupgrade](https://user-images.githubusercontent.com/30562547/53349696-95183980-3915-11e9-9605-7c19b2e56f5d.png)